consumer/consumer.h

```diff
@@ -0,0 +1,26 @@
+#ifndef CONSUMER_H
+#define CONSUMER_H
+
+#include "message_queue.grpc.pb.h"
+#include <grpcpp/grpcpp.h>
```

Comment:
I was thinking we can avoid external library declaration in headers right? This way when we try to run our client code in another machine we do not need to install the dependencies

Reply:
Static library banegi right? Toh does it matter?

Reply:
Headers file is also required for the client code. If header has links to external library then it will not work as it will expect external header files to be present in the system.

Reply:
Looks good